docs/grav.md

```diff
@@ -0,0 +1,41 @@
+## Grav has been deprecated. You can use the new [bus package](https://github.com/suborbital/e2core/tree/main/bus), which is a drop-in continuation of this project. 
```

Comment:
It is a bit weird that we say "Grav has been deprecated", then go on and [link](https://github.com/suborbital/e2core/tree/main/bus) to a page that has a giant GRAV logo and a readme that talks about Grav althroughout.

I understand that this may not be the case indefinitely and eventually `e2core bus` may get a proper re-branding but maybe we could just say something like "Grav is no longer maintained as a separate project" or "in this repository", but instead it "lives on" as part of the bus package of the e2core project?

Reply:
👍 Yeah, looks like we should update the e2core/bus README to be a little less "Grav!"-y

Reply:
I can see that. I just grabbed what Connor had written on the readme, and I think maybe the stronger wording helps to send the message that Grav isn't being maintained for external use, but for sure I think it needs a full `e2core bus` rebrand.